test/addons/csi-addons/kustomization.yaml

```diff
@@ -3,10 +3,10 @@
 
 ---
 resources:
-  - ${base_url}/crds.yaml
-  - ${base_url}/rbac.yaml
-  - ${base_url}/setup-controller.yaml
+  - https://raw.githubusercontent.com/csi-addons/kubernetes-csi-addons/v0.8.0/deploy/controller/crds.yaml
+  - https://raw.githubusercontent.com/csi-addons/kubernetes-csi-addons/v0.8.0/deploy/controller/rbac.yaml
+  - https://raw.githubusercontent.com/csi-addons/kubernetes-csi-addons/v0.8.0/deploy/controller/setup-controller.yaml
```

Comment:
Lets separate the change to use kustomization from the change to allow caching. See how it was done in https://github.com/RamenDR/ramen/pull/1346/commits/2d9fc24cb2fd93eece85c1f43d0f9bff45d4856d